src/components/Dashboard/DateTile.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const TileContainer = styled.div`
+  height: 130px;
+  width: 270px;
+  background-color: #296157;
+  box-shadow: 2px 2px gainsboro;
+  padding: 15px;
+  border-radius: 10px;
+  text-align: left;
+`;
+
+const DateText = styled.div`
+  color: white;
+  font-size: 30px;
+  margin: 10px;
+`;
+
+const TimeText = styled.div`
+  color: white;
+  font-size: 30px;
+  margin: 20px 10px;
+`;
```

Comment:
can the stylings start going in a separate file called `styled.tsx`?
this will help keep things organized.

Reply:
see discord advise on organization

Reply:
moved to separate file. Going to reorganize everything in another pr